File: gr00t/Isaac-GR00T/examples/H20/robots/sdk_package/example_py/h10-w/system/clear_error.py
import humanoid_sdk_py
import humanoid_sdk_py.h10w as h10w
import time

h10w_system = h10w.H10wSystem()
time.sleep(1)

print("=== 清除所有报错信息参数示例 ===")
ret = h10w_system.clearError()
if ret == 0:
    print("清除报错信息成功")
else:
    print("清除报错信息失败")

File: gr00t/Isaac-GR00T/examples/H20/robots/sdk_package/example/h10-w/system/clear_error.cpp
// clear_error.cpp

#include <iostream>
#include <iomanip>
#include "h10-w/h10w_system.h"
#include "common/common_def.h"

using namespace humanoid;
using namespace humanoid::h10w;

int main() {
  std::cout << "=== 清除所有报错信息参数示例 ===" << std::endl;
  H10wSystem system;
  std::cout << "清除报错信息" << (system.clearError() == 0 ? "成功" : "失败")
            << std::endl;
  return 0;
}